### ISSUE TITLE
[ES] Display the mapping of controllers (with images of controllers)

### ISSUE THREAD

**Issue by Feufochmar** [recalbox/recalbox-os#930]
Displaying the mapping (or expected mapping) of controllers with images (instead of text) would be nice to have for the following reasons:
- when configuring a controller, showing the expected controller is better than telling that it should follow the snes button layout (nobody reads huge blocks of texts, and not everybody know the snes layout)
- when using an emulator for a system using a different layout than the snes (megadrive for instance), showing how buttons are mapped is very welcomed (this image could be shown in the menu of the system, or in an help entry)

**Comment by substring** [comment]
yup, already had a similar idead for the mapping a few months ago before i joined the project. I lost my work since, but ... wait & see ;)

Concerning the emulator ..., i'm afraid it will just overload the display. Up do @digitalLumberjack to decide
